qp_qiime2/qp_qiime2.py

```diff
@@ -148,8 +152,13 @@
     ('beta', 'metric'): BETA_DIVERSITY_METRICS,
     ('alpha_phylogenetic', 'metric'): ALPHA_DIVERSITY_METRICS_PHYLOGENETIC,
     ('beta_phylogenetic', 'metric'): BETA_DIVERSITY_METRICS_PHYLOGENETIC,
+    # alpha rarefaction has some "forbidden" metrics and we need to be sure to
+    # not add them here
     ('alpha_rarefaction', 'metrics'): {
-        **ALPHA_DIVERSITY_METRICS, **ALPHA_DIVERSITY_METRICS_PHYLOGENETIC},
+        name: metric for name, metric in
+        {**ALPHA_DIVERSITY_METRICS,
+         **ALPHA_DIVERSITY_METRICS_PHYLOGENETIC}.items()
+        if metric not in alpha_rarefaction_unsupported_metrics},
```

Comment:
If these are all sets, the following should work, I think.

```suggestion
        ((ALPHA_DIVERSITY_METRICS |
          ALPHA_DIVERSITY_METRICS_PHYLOGENETIC) - alpha_rarefaction_unsupported_metrics),
```

Reply:
They are dicts ...

Reply:
Got it, thanks!